libdnf/rpm/solv_query.cpp

```diff
@@ -1191,6 +1335,239 @@ void SolvQuery::Impl::filter_reldep(Id libsolv_key, libdnf::sack::QueryCmp cmp_t
     }
 }
 
+void SolvQuery::Impl::filter_nevra(
+    const Nevra & pattern,
+    bool cmp_glob,
```

Comment:
Confusing. The value can be deduced from `cmp_type`. It is probably optimization (allways `true` in `resolve_pkg_spec()`.) The method is internal, so you can leave it as is now.
Can you add comment to `cmp_glob` argument? It is optimization and `cmp_glob` must be in sync with `cmp_type` argument.

Reply:
The comment was added.